### DISCUSSION TITLE
[Feature request] Palette chooser GUI

### DISCUSSION THREAD

**Discussion by commie-caden** [ghostty-org/ghostty#3971] [Feature Requests, Ideas] [answered]
## Description

Regarding the Linux/Adwaita mode. The default palette is decent but not my favorite. I would like a simple GUI appearance menu similar to what Ptyxis does:

![Screenshot From 2024-12-29 16-46-23](https://github.com/user-attachments/assets/e198b377-9a91-4121-8710-0ceac65e769a)

You could even take this a step further allowing users to import and export palettes with a config editor so we can customize graphically and interactively instead of with a config file and reloading the config repeatedly. Perhaps design-wise you could have a small preview of text to help the user visualize.

**Comment by edrozenberg** [comment]
For now the closest is this web configurator, https://ghostty.zerebos.com/settings/colors, after making any changes the text config will show up in https://ghostty.zerebos.com/app/import-export

**Comment by 00-kat** [comment, marked as answer]
Probably a part of #2354.
